configs/mirfm_config.yaml

```diff
@@ -0,0 +1,85 @@
+# ROBOT CONFIG =================================================================
+# To init MiR robot REST APIs and RMF states
+
+mirfm:
+  base_url: 'http://some.ip.address/api/v2.0.0'
```

Comment:
Nit: Technically it supports url as well.

Reply:
[520d2ca](https://github.com/osrf/fleet_adapter_mir/pull/27/commits/520d2cab116e91ee03871b0263157cc94e0e5b33)